packages/domain-events/src/Domain/Event/SubscriptionRefundedEventPayload.ts

```diff
@@ -2,6 +2,7 @@ import { SubscriptionName } from '@standardnotes/auth'
 
 export interface SubscriptionRefundedEventPayload {
   userEmail: string
-  subscriptionName: SubscriptionName
+  fromSubscriptionName: SubscriptionName
+  toSubscriptionName: SubscriptionName
```

Comment:
hmm if refunding we are cancelling a subscription so to say ... so no 'toSubscriptio` is needed right?

Reply:
Same as above, the toSubscriptionName should be CorePlan. Does that make sense?

Reply:
When someone cancels a core subscription they can get back to a basic user with no subscription

Reply:
@karolsojko come to think of it, it's the same case as above but reversed, so should be optional I think? A user can have for example a Plus subscription, buy a Pro subscription and then refund it. In that case, the user would go from Pro to Plus. Is that correct?

Reply:
@karolsojko actually I just realized we may not need this at all, how about we just add the new user role when a subscription is purchased and remove it if it's refunded? That way if a user doesn't have a subscription and a Pro plan is purchased for example, the user would have both the BasicUser and ProUser roles, but I think that'd be okay. Wdyt?

Reply:
Made the described change on the auth PR so will close this for now, will re-open if necessary 🙂